stories/Button/Button.story.jsx

```diff
@@ -105,20 +106,56 @@ const exampleTab = (
   </Tab>
 );
 
+const socialExampleTab = (
+  <Tab title="Example">
+    <StoryContainer>
+      <Container fluid>
+        <Title as="h2">Google button</Title>
+        <Example
+          component={<SocialButton provider="google" title="google-button" />}
+          code="<SocialButton provider='google' title='google-button'/>"
+        />
+
+        <Title as="h2">Facebook button</Title>
+        <Example
+          component={
+            <SocialButton provider="facebook" title="facebook-button" />
+          }
+          code={"<SocialButton provider='facebook' title='facebook-button'/>"}
+        />
+      </Container>
+    </StoryContainer>
+  </Tab>
+);
+
 const description = `Buttons express what action will occur when the user clicks
```

Comment:
how about we update this description to this specific buttons?

like `this buttons are used to login access with its respective social media.`

Reply:
Will they be used only for login?

Reply:
Good Question. We need to confirm this info with Arthur. 👍